.env

```
@@ -1,3 +1,4 @@
 ## API
 DATABASE_URL=postgresql://stationapi:stationapi@localhost/stationapi
-DISABLE_GRPC_WEB=false
\ No newline at end of file
+DISABLE_GRPC_WEB=false
+DISABLE_BUS_FEATURE=false
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)